dynamic-plugins/imports/package.json

```diff
@@ -22,7 +22,7 @@
     "@janus-idp/backstage-plugin-bulk-import": "1.6.0",
     "@janus-idp/backstage-plugin-bulk-import-backend": "4.0.0",
     "@janus-idp/backstage-plugin-jfrog-artifactory": "1.9.0",
-    "@janus-idp/backstage-plugin-keycloak-backend": "3.0.0",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.13.4",
```

Comment:
I believe that we can leave this as `3.0.0` and just close out this PR since you have this [one](https://github.com/janus-idp/backstage-showcase/pull/1823) already open

Reply:
@PatAKnight, "@janus-idp/backstage-plugin-keycloak-backend" with version "3.0.0" - bad release without dist-dynamic folder, so keycloak-backend doesn't work at this version.

Reply:
See more https://www.npmjs.com/package/@janus-idp/backstage-plugin-keycloak-backend?activeTab=code . There is not dist-dynamic folder.

Reply:
Oh, we don't have imports folder any more...